apps/src/gamelab/GameLab.js

```diff
@@ -970,6 +995,13 @@ GameLab.prototype.getAnimationMetadata = function () {
   return this.studioApp_.reduxStore.getState().animations;
 };
 
+/**
+ * Get metadata for a particular animation.
+ */
+GameLab.prototype.getAnimationMetadataByKey = function (key) {
+  return this.getAnimationMetadata().filter(animation => animation.key === key)[0];
```

Comment:
You can use `Array.find` in es6 to avoid the index 0 lookup: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find